src/main/java/bio/terra/cli/command/resources/Describe.java

```diff
@@ -2,35 +2,73 @@
 
 import bio.terra.cli.command.helperclasses.BaseCommand;
 import bio.terra.cli.command.helperclasses.FormatOption;
-import bio.terra.cli.context.CloudResource;
 import bio.terra.cli.service.WorkspaceManager;
+import bio.terra.workspace.model.ResourceDescription;
 import picocli.CommandLine;
-import picocli.CommandLine.Command;
 
 /** This class corresponds to the third-level "terra resources describe" command. */
-@Command(name = "describe", description = "Describe an existing controlled resource.")
+@CommandLine.Command(name = "describe", description = "Describe a resource.")
 public class Describe extends BaseCommand {
-
   @CommandLine.Option(
       names = "--name",
       required = true,
-      description = "The name of the resource, scoped to the workspace.")
+      description = "Name of the resource, scoped to the workspace.")
   private String name;
 
   @CommandLine.Mixin FormatOption formatOption;
 
-  /** Describe an existing controlled resource. */
+  /** Describe a resource. */
   @Override
   protected void execute() {
-    CloudResource resource =
-        new WorkspaceManager(globalContext, workspaceContext).getControlledResource(name);
+    ResourceDescription resource =
+        new WorkspaceManager(globalContext, workspaceContext).getResource(name);
     formatOption.printReturnValue(resource, Describe::printText);
   }
 
   /** Print this command's output in text format. */
-  private static void printText(CloudResource returnValue) {
-    OUT.println("Name: " + returnValue.name);
-    OUT.println("Type: " + returnValue.type);
-    OUT.println("Cloud Id: " + returnValue.cloudId);
+  public static void printText(ResourceDescription returnValue) {
+    OUT.println("Name:         " + returnValue.getMetadata().getName());
```

Comment:
Probably not this PR, but if we're going to do whitespace pretty printing, it's probably worth doing it programmatically instead of hard coding the amount of whitespace to pad by.

Reply:
Agreed. Added to the laundry list of style-related improvements [here](https://broadworkbench.atlassian.net/browse/PF-574).